lib/Doctrine/ODM/PHPCR/UnitOfWork.php

```diff
@@ -927,16 +914,22 @@ public function scheduleMove($document, $targetPath)
 
         switch ($state) {
             case self::STATE_NEW:
-                unset($this->scheduledInserts[$oid]);
```

Comment:
this is a behaviour change i think, if we previously auto-persisted documents upon move operations.